src/teleop/gui/src/components/Raman.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <h3>Raman</h3>
+    <ToggleButton
+      :current-state="ramanLaserState"
+      label-enable-text="Raman Laser On"
+      label-disable-text="Raman Laser Off"
+      @change="toggleRamanLaser()"
+    />
+  </div>
+</template>
+
+<script>
+import ToggleButton from "./ToggleButton.vue";
+import ROSLIB from "roslib";
+
+export default {
+  components: {
+    ToggleButton,
+  },
+  data() {
+    return {
+      ramanLaserState: false,
+      ramanService: null,
+    };
+  },
+
+  created: function () {
+    this.ramanService = new ROSLIB.Service({
+      ros: this.$ros,
+      name: "change_arm_laser_state",
```

Comment:
Needs to be changed to change_raman_laser_state, had some interfacing confusion here